RELEASE/scripts/autoscend/auto_tower.ash

```diff
@@ -54,7 +55,17 @@ boolean L13_towerNSContests()
 			case 1:
 				acquireMP(160); // only uses free rests or items on hand by default
 
-				provideInitiative(400, true);
+				if (auto_have_familiar($familiar[Oily Woim]))
+				{
+					use_familiar($familiar[Oily Woim]);
+				}
+
+				autoMaximize("initiative -equip snow suit", 1500, 0, false);

```

Comment:
1. why 1500 and not 400
2. I am looking at the function now (I wanted to check if there is a familiar switching option baked in, there is not). And if I am reading it correctly putting that false in here means it will do a simulation instead of actually use it. true will use it, false will simulate.

Reply:
you've got things backwards: `true` means simulate

Reply:
hmmm, ok. I see it now.
the code it calls is
```
boolean autoMaximize(string req, boolean simulate)
{
	if(!simulate)
	{
		debugMaximize(req, 0);
		tcrs_maximize_with_items(req);
#		user_confirm("Beep");
	}
	return maximize(req, simulate);
}
```
if you send it false, if(!simulate) flips it into a true, meaning that the if function is called and it performs a debugMaximize

if you send it a true, !simulate flips it into a false, meaning the if function is not called and it does not call debufMaximize.

regardless of whether debugMaximize was called or not, a mafia built in maximize is then called with an unflipped simulate bool

So a false means: debugMaximize and then run mafia maximize
while a true means: do not run debugMaximize and then perform a mafia simulation

know why it is doing it this way?